lib/interfaces/common.js

```diff
@@ -165,6 +165,7 @@ module.exports = function(suites, context, mocha) {
        * @returns {*}
        */
       only: function(mocha, test) {
+        if (mocha.options.forbidOnly) throw new Error('`.only` forbidden');
```

Comment:
we're throwing a similar error here and [here](https://github.com/mochajs/mocha/blob/master/lib/runner.js#L527).  there's precedence for using [`createUnsupportedError`](https://github.com/mochajs/mocha/blob/master/lib/errors.js#L61) with checks for `this.skip()`, so instead of `throw new Error()`, please use `throw createUnsupportedError()`

Reply:
I changed the usage to `createUnsupportedError()` in `common.js` and `runner.js`